Exchange-Guide.md

```diff
@@ -2,198 +2,14 @@
+`xud` is in alpha stage and this guide aims to give exchange operators a 'feel' of how things will work. `xud` in it's current stage should never be connected to a production system or be configured for mainnet usage. The `xud-simnet` uses test coins at all times. We are working on an improved setup using docker, which will also be the preferred way to use `xud` for exchanges in production.
 
-## Describing the Setup
+### XUD SimNet
 
-To connect to `xud-simnet` you'll need to run several components which together provide support for exchanging order information and atomic swaps between BTC and LTC on the lightning network. You will setup the following components:
+Follow [this guide](SimNet-Guide.md) to set up the XUD SimNet. Once the setup of `xud-simnet` is completed, you will be able to query pending orders, place orders, and experiment with `xud`'s rich set of commands. This guide is written for Linux & MacOS operating systems and geared towards IT administrators of exchanges.
 
-- `BTCD` - full node, connected to the BTC chain
-- `LTCD` - full node, connected to the LTC chain
-- `LND-BTC` - lightning network daemon for the BTC network
-- `LND-LTC` - lightning network daemon for the LTC network
-- `XUD` - Exchange Union's decentralized exchange layer
-
-## Installation 
-
-### Prerequisites
-
-Make sure you have the below programs installed:
-- [git](https://gist.github.com/derhuerst/1b15ff4652a867391f03#file-linux-md)
-- [make](https://www.cyberciti.biz/faq/debian-linux-install-gnu-gcc-compiler/)
-- [node.js + npm](https://nodejs.org/en/download/package-manager/#debian-and-ubuntu-based-linux-distributions)
-- python2.7 (preinstalled on most systems, check with `python --version`)
-- killall (`sudo apt-get install psmisc`)
-
-### Installing Go
-
-If Go is not yet installed on your system please use the following command to install it:
-
-```
-sudo apt-get install golang-1.10-go
-```
-
-If Go is not included in your package manager, follow the official [install instructions](https://golang.org/doc/install). Note that golang-1.10-go puts binaries in /usr/lib/go-1.10/bin. If you want them on your PATH, you need to make that change yourself. Alternatively, you can create a softlink as follows:
-
-```
-sudo ln -s /usr/lib/go-1.10/bin/go /usr/local/bin/go 
-```
-
-### Repository Checkout
-
-```
-git clone https://github.com/ExchangeUnion/xud-simnet.git ~/xud-simnet
-```
-
-This should create the `xud-simnet` directory under your home directory.
-
-### Setup ~/.bashrc
-
-To enable access to the scripts please update and source your `.bashrc`:
-
-```
-source ~/xud-simnet/setup.bash
-source ~/.bashrc
```

Comment:
Maybe `source ~/.bashrc` should run before `source ~/xud-simnet/setup.bash`?

Reply:
Already fixed.